projects/bazel-rules-fuzzing-test/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+
+RUN apt-get update && apt-get -y install \
+    python wget
+
+# Install Bazelisk
+RUN wget -O /usr/local/bin/bazel https://github.com/bazelbuild/bazelisk/releases/download/v1.7.4/bazelisk-linux-amd64
```

Comment:
This needs to go in https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/Dockerfile so that projects dont need to install it themselves and we give a consistent version.
also, use 
ENV BAZEL_VERSION 1.7.4
RUN wget -O /usr/local/bin/bazel https://github.com/bazelbuild/bazelisk/releases/download/v$BAZEL_VERSION/bazelisk-linux-amd64

Reply:
Sounds good - let me do all this in a separate PR.

Reply:
Addressed in #4976.